### ISSUE TITLE
server_response some times contains a " in content breaking json parsing: "server_response": "ÈÝ$Þ³Pjpae5àÝËùý©t"9",

### ISSUE THREAD

**Issue by BillBinAz** [jimboca/udi-poly-airscape#4]
From the log:  
```
2019-10-27 14:14:56,192 [Controller] [ERROR] Airscape whf5300:reponse:get: Failed to convert json {    "fanspd": 0,    "doorinprocess": 0,    "timeremaining": 0,    "macaddr": "60:CB:FB:00:80:3F",    "ipaddr": "192.168.0.23",    "model": "5300 WHF",    "softver": "2.17.1",    "interlock1":0,    "interlock2": 1,    "cfm": 0,    "power": 0,    "inside":-99,    "attic": 63,    "oa":-99,    "server_response": "ÈÝ$Þ³Pjpae5àÝËùý©t"9",    "dip_switches": "11110",    "remote_switch": "1111",    "setpoint": 0,    "dns": "192.168.0.1"    }: Expecting ',' delimiter: line 1 column 332 (char 331)
Traceback (most recent call last):
  File "/home/pi/.polyglot/nodeservers/Airscape/pgSession.py", line 80, in response
    rdata = json.loads(rtxt)
```
Ended up putting in a bandaid hack in pgSession.py

```
                rtxt = remove_control_characters(response.text)
                # cleanup will leave an extra " in the content
                start = rtxt.find("\"server_response\"")
                end = rtxt.find("\"dip_switches\"")
                rtxt = rtxt.replace(rtxt[start:end], "")
                rdata = json.loads(rtxt)

```

Not pretty.  If I find a good way to pair up the " in regex will try that.

**Comment by jimboca** [comment]
Oh damn, that's ugly, but really appreciate your fix!

**Comment by jimboca** [comment]
But, just thinking about it, it may best to just do a regexp to change:
"server_response": "ÈÝ$Þ³Pjpae5àÝËùý©t"9", 
to
"server_response": "",
and not do the remove_control_characters.

I asked Airscape about that but never really got someone who understood and gave up.

**Comment by BillBinAz** [comment]
I did the same, and never got a good reply why either.

**Comment by BillBinAz** [comment]
As far as a regex goes, I have no confidence that a unescaped " followed by the , will not occur in the server response.  So just just took advantage of the fixed attribute order list and removed it entirely.  It seems both the JSON and XML are built using simple str concat with any any escaping at all.

**Comment by jimboca** [comment]
Fixed in 2.0.9
